xbmc/Application.cpp

```diff
@@ -2939,6 +2939,36 @@ void CApplication::Stop(int exitCode)
   Sleep(200);
 }
 
+bool CApplication::ResumePlayer(const CFileItemPtr& itemIn) {
+	CLog::Log(LOGDEBUG, "CApplication::ResumePlayer resume item %s", itemIn->GetPath().c_str());
+	if (m_pPlayer->HasPlayer())
```

Comment:
I did add this because I know that the player termination sometimes did not get done during onSleep and will go on during wakeup. Not sure about the need of this.